com.abagames.rrotage.json

```diff
@@ -0,0 +1,164 @@
+{
+  "app-id": "com.abagames.rRootage",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
```

Comment:
Have you tried this against 18.08? 1.6 is out of security support and I'd encourage trying to use the more modern version if possible.